README.md

```diff
@@ -5,18 +5,22 @@
 - Name: usda-cdl
 - Package: `stactools.usda_cdl`
 - PyPI: https://pypi.org/project/stactools-usda-cdl/
-- Owner: @pholleway
-- Dataset homepage: http://example.com
+- Owner: @gadomski, @pholleway
+- Dataset homepage: https://www.nass.usda.gov/Research_and_Science/Cropland/Release/index.php
 - STAC extensions used:
+  - [classification](https://github.com/stac-extensions/classification)
```

Comment:
We should add the raster extension to assets -- opened https://github.com/stactools-packages/usda-cdl/issues/3 to capture.